decryptor/postgresql/pg_decryptor.go

```diff
@@ -408,6 +408,41 @@ func (proxy *PgProxy) sendClientError(msg string, logger *log.Entry) error {
 	return nil
 }
 
+// stopClientThread sends a signal to a client thread to stop. Returns error in
+// case of an error or timeout. Is used to stop and reload client with TLS
+func (proxy *PgProxy) stopClientThread(logger *log.Entry) error {
```

Comment:
maybe lets name `stopClientGoroutine` or `stopProxyClientConnection` as opposite to `ProxyClientConnection`?